app/services/sign_in/user_loader.rb

```diff
@@ -23,6 +24,7 @@ def reload_user
       user_identity.uuid = access_token.user_uuid
       current_user.uuid = access_token.user_uuid
       current_user.last_signed_in = session.created_at
+      current_user.fingerprint = @request_ip
```

Comment:
Can you add `request_ip` to the `attr_reader` and then refer to this without the `@`?